### PR TITLE
Add changelog entry for v0.2.1

### PR DESCRIPTION
**Pull request by santisoler**
I forgot to update the new DOI for version v0.2.0 on the Zenodo badge in `README.rst` and on the instructions in `CITATION.rst`. These new release just updates the DOI for v0.2.0. It does not issue a new DOI because there are no changes on code nor documentation.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to the docstring, gallery, and/or tutorials.

### CODE REVIEW

**Comment by santisoler** [comment]
@leouieda As you could see, I forgot to update the DOI, so I thought it would be better to make a new release just to solve this issue. Besides, I've opened a PR (fatiando/contributing#11) for adding instructions to do so on `MAINTENANCE.md`.

Do you think it's necessary? Should we create a new version on Zenodo with a different DOI or keep the latest one for v0.2.0?

**Comment by leouieda** [comment]
Hi @santisoler that's fine and we don't need to archive this package on zenodo as well. To avoid this issue in the future, we can use the DOI for all versions: 10.5281/zenodo.3086001 That will automatically redirect to the latest one. If anyone wishes to cite a specific version they can still do so but it will make our lives easier. What do you think?

**Comment by santisoler** [comment]
I think I don't have a clear idea of what's the right way to do this. I thought it's better to generate a new DOI for each release, but maybe it's not completely necessary. Moreover, what happens when we publish a library on JOSS? Should we keep telling people to cite it through the JOSS DOI even if they are using a newer version?

**Comment by santisoler** [comment]
For this specific case I agree it's not necessary to issue a new DOI for it, so I think we can merge this PR and make the new release.

**Comment by leouieda** [comment]
> For this specific case I agree it's not necessary to issue a new DOI for it, so I think we can merge this PR and make the new release.

:+1: yep, agreed.

Thanks for starting the conversation on Slack. This is an unsolved problem and I don't think there a clear way to do this. But we can experiment and change tactics later on in case we find a better way.

**Comment by leouieda** [comment]
The important thing is to not get frozen in place until we find a solution (which could take a really long time). There isn't anything inherently wrong with the way we've been doing things.

File: CITATION.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ that goes into building and maintaining this project.
 
 If you used RockHound in your research, please consider citing it:
 
-   Uieda, L., and Soler, S.R. (2019). Rockhound: Download geophysical models/datasets
-   and load them in Python (Version 0.1.0). Zenodo. https://doi.org/10.5281/zenodo.3086002
+   Uieda, L., and Soler, S.R., Pesce, A. (2020). Rockhound: Download geophysical
+   models/datasets and load them in Python (Version 0.2.0). Zenodo.
+   https://doi.org/10.5281/zenodo.3627166
```
Comment (leouieda) [inline CITATION.rst:11]: We could replace this part by:

If you used RockHound in your research,  please consider citing the Zenodo archive: https://doi.org/10.5281/zenodo.3086001

This DOI redirects to the latest release at the time. DOIs for specific releases can be found under "Versions" on the Zenodo site. Use one of these if you wish to refer to a specific version.

**Pull request closed without merging**